addons/im_livechat/models/chatbot_script_question_answer.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from odoo.osv import expression
+
+
+class ChatbotScriptQuestionAnswer(models.Model):
+    _name = 'im_livechat.chatbot.script_question_answer'
+    _description = 'Chatbot Script Question Answer'
+
+    name = fields.Char(string='Name')
+    sequence = fields.Integer(string='Sequence')
+    step_id = fields.Many2one(
+        'im_livechat.chatbot.script_step', string='Script Step', required=True, ondelete='cascade')
+    chatbot_id = fields.Many2one(related='step_id.chatbot_id', store=True)
+
+    # TODO PKO: Test if conflict with name_get ? not sure why we defined name_get in the first place...
+    def _compute_display_name(self):
+        for record in self:
+            record.display_name = record.name
+
+    def name_get(self):
+        return [(
+            answer.id, "%s: %s" % (answer.step_id.message, answer.name)
```

Comment:
Side-note: what happens if the message is on several lines?

Tried to test but got some strange behavior:

![image](https://user-images.githubusercontent.com/41282536/153217063-0b67fd7c-94a3-472f-b985-b9b784055cef.png)

![image](https://user-images.githubusercontent.com/41282536/153217135-551e1458-73c2-4fd6-88ab-0d6e50716992.png)

![image](https://user-images.githubusercontent.com/41282536/153217187-2c810578-e9e2-43ae-8ddd-630760495b28.png)

Can you double check?
Maybe we should have a separate "label" / "title" field for questions, that would be use here? 
(To check with Luc if unsure).